OptimizelySDKCore/OptimizelySDKCore/OPTLYProjectConfig.h

```diff
@@ -51,9 +51,11 @@ NS_ASSUME_NONNULL_END
 /// List of Optimizely Experiment objects
 @property (nonatomic, strong, nonnull) NSArray<OPTLYExperiment *><OPTLYExperiment> *experiments;
 /// List of Optimizely Event Type objects
-@property (nonatomic, strong, nonnull) NSArray<OPTLYEvent *><OPTLYEvent> *events;
-/// List of audience ids
-@property (nonatomic, strong, nonnull) NSArray<OPTLYAudience *><OPTLYAudience> *audiences;
+@property (nonatomic, strong, nonnull) NSArray<OPTLYEvent> *events;
+/// List of audience objects
+@property (nonatomic, strong, nonnull) NSArray<OPTLYAudience> *audiences;
```

Comment:
Please fix these declarations to conform to JSONModel parsing (i.e @property (nonatomic, strong, nonnull) NSArray<OPTLYAudience *><OPTLYAudience> *audiences;)